documentation/src/main/asciidoc/topics/con_clients_marshalled_objects.adoc

```diff
@@ -0,0 +1,24 @@
+[id='clients-marshalled-objects_{context}']
+= Clients and marshalled objects
+If you configure cache encoding to store marshalled objects, then you also need to configure clients to operate on those caches.
+
+[discrete]
+== Hot Rod clients
+
+The media type you use to encode caches should match the client marshaller.
+For example, if you configure cache encoding with the `application/x-protostream` media type, you should use the ProtoStream marshaller.
+
+Likewise, if you use `JavaSerializationMarshaller` or `GenericJBossMarshaller` you should encode caches with the corresponding media type.
```

Comment:
I think we should be explicit here and say something like:

"encode caches with the `application/x-java-serialized-object` or `application/x-jboss-marshalling` media type, respectively.